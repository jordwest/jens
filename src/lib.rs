extern crate pest;
#[macro_use]
extern crate pest_derive;

pub use block::{Block, IteratorLocation, Line, LineSegment};
pub use grammar::File;

mod block;
mod grammar;
