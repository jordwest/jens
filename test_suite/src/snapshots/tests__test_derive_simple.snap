---
created: "2019-02-17T03:31:59.641635Z"
creator: insta@0.6.2
source: src/lib.rs
expression: t2.to_string()
---
This template has two placeholders:
  One: "hello!"
  Two: "Hullo"
